egs/an4/tts1/run.sh

```diff
@@ -182,38 +144,7 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
 fi
 
 if [ -z ${tag} ];then
```

Comment:
Could you add a space after semicolon (in all run.sh files)?

Reply:
Fixed.